publisher/product/ontology/build.sh

```diff
@@ -495,6 +496,51 @@ function ontologyBuildIndex () {
 	return $?
 }
 
+function ontologyCreateMergedFiles() {
+
+  require source_family_root || return $?
+  require tag_root || return $?
+
+  logStep "ontologyCreateMergedFiles"
+
+  pushd "${tag_root:?}" >/dev/null || return $?
+
+  local -r maxParallelJobs=1
+  local numberOfParallelJobs=0
+
+  log "Running ${maxParallelJobs} 'merge-imports' jobs in parallel:"
+
+  # include only the RDF files with the ontologies listed in the source "catalog-v001.xml"
```

Comment:
@mereolog whether the list of ontologies for which _Merged_ files are to be generated is to be based on:

1. _owl: imports_ in the [DEV_SPEC](https://github.com/edmcouncil/ontology-publisher/blob/6be7550b756b384c5323fbd1a79e21cbb854cadb/README.md?plain=1#L88) file
2. _owl: imports_ in the [PROD_SPEC](https://github.com/edmcouncil/ontology-publisher/blob/6be7550b756b384c5323fbd1a79e21cbb854cadb/README.md?plain=1#L89) file
3. all ontologies listed in the `catalog-v001.xml` file (this is currently the case)
4. all _.rdf_ files in a given ontology repository

Reply:
Go for option 4 (with the filtering described in https://github.com/edmcouncil/ontology-publisher/pull/104#discussion_r918692230 below).